atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -509,7 +509,8 @@ def add_log_type(output):
                     'parent_id': parent_id,
                     'repositories': repositories,
                     'tags': list(tags),
-                    'config': config
+                    'config': config,
+                    'autorebuild': is_rebuild(self.workflow)
```

Comment:
Good point, it should probably be a build property not an output property. What should the key be then?